README.md

```diff
@@ -70,7 +71,49 @@ Before beginning a sprint:
+After finishing a sprint we hold a retrospective to discuss what went well, what didn't go so well, and what we decide to do differently next sprint. Part of retrospective involves examining our [burndown chart](https://en.wikipedia.org/wiki/Burn_down_chart) to gauge the accuracy of our estimates and inform how we estimate and commit to the next sprint's tasks. Retrospective is also the perfect time to adjust the process to better suit our needs.
+
+## Velocity Tracking
+
+### What is Velocity Tracking?
+
+Velocity tracking is the practice of using our average historical output to inform how we plan and commit to upcoming work. 
+
+Velocity is a number that represents the amount of work completed by a team during a specific time period. Typically, we talk about “sprint velocity” and “average sprint velocity”. `sprint-velocity` describes the amount of work completed in a single sprint. `average-sprint-velocity` is an average of the previous *n* sprints (where *n* is something like 2 - 4 sprints). We use the latter during sprint planning to determine how much work to commit to in the sprint being planned.
+
+### How Do We Track Velocity?
+
+Velocity tracking stuff mostly occurs during sprint planning. 
+
+If the sprint you're planning is the first sprint where you'll be incorporating anything related to velocity you need to choose an initial "fake" `average-sprint-velocity`. Its value might be something like 0.5 (avg. number of points completed per person per hour).
+
+If you just finished a sprint and all tasks in that sprint were assigned a point value, then you need to calculate that sprint's `sprint-velocity` and use that to calculate `average-sprint-velocity`. You do this like so:
+
+> *total person-hours* = The sum of the hours each team member contributed to the sprint
+
+> *total number of points completed* = The sum of the point values of tasks that are 100% “done”
+
+> **sprint-velocity** = *total number of points completed* / *total person-hours*
+
+> **average-sprint-velocity** = average **sprint-velocity** of previous *n* sprints
+
+*Note: When calculating `total person-hours` aka each team member's availability for the sprint you're planning. For example, if your sprint is 2 weeks of 5 8-hour days then each person has a max of 80 available hours. But you might reduce this number if a company holiday falls within the sprint, if a team member is taking time off, or generally if someone's capacity is reduced for any other reason.* 
+
+*Note: Dividing the total points by the total hours results in the smallest possible unit of one point, per one person, per one hour aka `sprint-velocity`. Since `sprint-velocity` is broken down to the smallest unit, it can scale well with any number of team members as well as any number of days in a sprint.*
```

Comment:
Notes are easy to skip over because they seem unimportant. A note should either be super short or not a note. These notes also don't have too much context at this point in the "story" that the person is reading. I imagine at this point both numbers don't have too much meaning to the person.

Also to go along with my comment above, having that succinct relationship between the two values (while recognizing that they're very distinct) will be good to start. Then we have three sections below with one section about calculating sprint-velocity, one section about calculating and then utilizing average-velocity, and then a final section about individual velocity (as it relates to average-velocity).